billingportal/configuration/client.go

```diff
@@ -0,0 +1,94 @@
+// Package configuration provides API functions related to billing portal configurations.
+package configuration
+
+import (
+	"net/http"
+
+	stripe "github.com/stripe/stripe-go/v72"
+	"github.com/stripe/stripe-go/v72/form"
+)
+
+// Client is used to invoke /billing_portal/configurations APIs.
+type Client struct {
+	B   stripe.Backend
+	Key string
+}
+
+// New creates a new billing portal configuration.
+func New(params *stripe.BillingPortalConfigurationParams) (*stripe.BillingPortalConfiguration, error) {
+	return getC().New(params)
+}
+
+// New creates a new billing portal configuration.
+func (c Client) New(params *stripe.BillingPortalConfigurationParams) (*stripe.BillingPortalConfiguration, error) {
+	configuration := &stripe.BillingPortalConfiguration{}
+	err := c.B.Call(http.MethodPost, "/v1/billing_portal/configurations", c.Key, params, configuration)
+	return configuration, err
+}
+
+// Iter is an iterator for billing portal configurations.
```

Comment:
I have a preference for this at the bottom to be consistent. I was thrown off by it. Doesn't really matter long term but would help with reviews